Makefile

```diff
@@ -344,13 +347,46 @@ telemetrygen:
 	cd ./cmd/telemetrygen && GO111MODULE=on CGO_ENABLED=0 $(GOCMD) build -trimpath -o ../../bin/telemetrygen_$(GOOS)_$(GOARCH)$(EXTENSION) \
 		-tags $(GO_BUILD_TAGS) .
 
+# helper function to update the core packages in builder-config.yaml
+# input parameters are 
+# $(1) = path/to/versions.yaml (where it greps the relevant packages)
+# $(2) = path/to/go.mod (where it greps the package-versions)
+# $(3) = path/to/builder-config.yaml (where we want to update the versions)
+define updatehelper
+	if [ ! -f $(1) ] || [ ! -f $(2) ] || [ ! -f $(3) ]; then \
+			echo "Usage: updatehelper <versions.yaml> <go.mod> <builder-config.yaml>"; \
+			exit 1; \
+	fi
+	grep "go\.opentelemetry\.io" $(1) | sed 's/^\s*-\s*//' | while IFS= read -r line; do \
+			if grep -qF "$$line" $(2); then \
+					package=$$(grep -F "$$line" $(2) | head -n 1 | awk '{print $$1}'); \
+					version=$$(grep -F "$$line" $(2) | head -n 1 | awk '{print $$2}'); \
+					builder_package=$$(grep -F "$$package" $(3) | awk '{print $$3}'); \
+					builder_version=$$(grep -F "$$package" $(3) | awk '{print $$4}'); \
+					if [ "$$builder_package" == "$$package" ]; then \
+						echo "$$builder_version";\
+						sed -i -e "s|$$builder_package.*$$builder_version|$$builder_package $$version|" $(3); \
+						echo "[$(3)]: $$package updated to $$version"; \
+					fi; \
+			fi; \
+	done
+endef
+
+
 .PHONY: update-otel
 update-otel:$(MULTIMOD)
 	$(MULTIMOD) sync -s=true -o ../opentelemetry-collector -m stable --commit-hash $(OTEL_STABLE_VERSION)
-	git add . && git commit -s -m "[chore] multimod update stable modules"
+	git add . && git commit -s -m "[chore] multimod update stable modules" ; \
 	$(MULTIMOD) sync -s=true -o ../opentelemetry-collector -m beta --commit-hash $(OTEL_VERSION)
-	git add . && git commit -s -m "[chore] multimod update beta modules"
+	git add . && git commit -s -m "[chore] multimod update beta modules" ; \
+	$(call updatehelper,$(CORE_VERSIONS),$(GOMOD),./cmd/otelcontribcol/builder-config.yaml) 
+	$(call updatehelper,$(CORE_VERSIONS),$(GOMOD),./cmd/oteltestbedcol/builder-config.yaml)
+	sed -i -E -e "5,6s/[0-9]+\.[0-9]+\.[0-9]/$(TRIMMED_VERSION)/" ./cmd/otelcontribcol/builder-config.yaml 
```

Comment:
the two sed's here are no longer needed correct?

Reply:
Yes you're absolutely correct since we don't need to update the config files head version as you mentioned.